src/components/views/messages/MImageBody.tsx

```diff
@@ -160,7 +160,7 @@ export default class MImageBody extends React.Component<IBodyProps, IState> {
     };
 
     private clearError = (): void => {
-        MatrixClientPeg.get().off(ClientEvent.Sync, this.reconnectedListener);
+        MatrixClientPeg.get()?.off(ClientEvent.Sync, this.reconnectedListener);
```

Comment:
Worth a comment why this and unmount don't use safe?

Reply:
We wouldn't want the clean-up to throw, I don't think its worth commenting though